### DISCUSSION TITLE
iOS simulator

### DISCUSSION THREAD

**Discussion by mukswilly** [eycorsican/leaf#254] [Q&A]
How can I run an app containing libleaf on an iOS simulator? This is because I always get the error : (building for iOS Simulator, but linking in object file built for iOS, for architecture arm64). I need to use the simulator to check out designs quickly plus SwiftUI previews require the simulator too.

**Comment by AFateh7** [comment]
@mukswilly 
I faced this same issue in the following configuration Xcode 13.1 + Monterey OS 12.0.1 + Intel chip  I had to go tot he Architectures section and replace $(ARCHS_STANDARD) by x86_64. I hope this helps you.

**Reply by mukswilly** [reply]
Hello, thanks for your help. Although your answer didn't directly solve my issue, It put me on the right path to figuring out the solution. The problem is that I am using an M1 MackBook. In addition to your answer, I had to set the Excluded Architectures and add "Any iOS simulator SDK - arm64". After that, In the Applications folder, I had to set Xcode to be opened using Rosetta and that was all. That said, I don't know if I should mark your reply as the answer but I am truly grateful for your help.
